app/client-v2/src/components/ProjectView/ProjectPage.vue

```diff
@@ -36,7 +47,7 @@ onUnmounted(() => {
     </div>
     <div class="surface-card p-4 shadow-2 border-round">
       <ProjectPostRun v-if="store.startedRun" />
-      <ProjectPreRun v-else />
+      <ProjectPreRun v-if="store.project && store.project.samples" />
```

Comment:
also why the change here? we either want to see postrun or pre run

Reply:
These changes were to try and get around some error I was getting - but now if I revert the changes, the tests all still pass, and the page still works. So I'm not sure what was going on, but we don't need these changes.

Reply:
Reverted!